src/main/java/frc/robot/Robot.java

```diff
@@ -308,7 +308,7 @@ private void spinControlPanel() {
       detectedColorEntry.setString(colorString);
       confidenceEntry.setDouble(match.confidence);
 
-      double doubleContolPanelSpinAmount = controlPanelSpinAmount * 2;
+      int doubleContolPanelSpinAmount = controlPanelSpinAmount * 2;
```

Comment:
Variable `doubleContolPanelSpinAmount` does not need to be of type `double` due to that `controlPanelSpinAmount` is an integer itself.